File: src/main/java/problems/Uri1014_Consumo.java
package problems;

import java.io.IOException;
import java.math.BigDecimal;
import java.math.RoundingMode;
import java.util.Scanner;

public class Uri1014_Consumo {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        int km = ler.nextInt();
        double gasosa = ler.nextDouble();
        double consumo;

        BigDecimal km1 = new BigDecimal(gasosa).setScale(1, RoundingMode.HALF_DOWN);
        gasosa = km1.doubleValue();
        consumo = km / gasosa;

        System.out.println(new BigDecimal(consumo).setScale(3, RoundingMode.HALF_DOWN) + " km/l");
    }
}

File: src/main/kotlin/problems/Uri1014_Consumo.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    val km = ler.nextInt()
    var gasosa = ler.nextDouble()
    val consumo = km / gasosa
    println("%.3f km/l".format(consumo))
}
